src/Analysis/GWASV2/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -7,8 +7,13 @@ const DismissibleMessage = ({
   title = 'Placeholder Title',
   description = 'placeholder description',
   messageType = 'success',
+  dismissMessage = null,
 }) => {
   const [open, setOpen] = useState(true);
+  const close = () => {
+    if (dismissMessage) dismissMessage();
```

Comment:
Is this conditional necessary? I think the props validation requires the function should this should always be true (I think? I don't think prop validation would accept a falsey function)